### PR TITLE
data observatory analysis

### PR DESCRIPTION
**Pull request by javisantana**
cc @javierarce

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/data-observatory-measure-model.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+var BaseAnalysisFormModel = require('./base-analysis-form-model');
+var template = require('./data-observatory-measure.tpl');
+var ColumnOptions = require('../column-options');
+var DataObservartoryMetadata = require('../data-observatory-metadata');
+
+/**
+ * Form model for a centroid and weighted-centroid analysis
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/data-observatory-measure-model.js]: I think we should get rid of this kind of comments so this mismatch doesn't happen :)

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/data-observatory-measure.tpl [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<form>
+  <div class="Editor-HeaderInfo">
+    <div class="Editor-HeaderNumeration CDB-Text is-semibold u-rSpace--m">2</div>
+    <div class="Editor-HeaderInfo-inner CDB-Text" data-fields="source,final_column">
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/data-observatory-measure.tpl]: If `final_column` depends on the output of the other fields, I'd move it to the bottom of the form.

**Comment by javierarce** [comment]
👍  just some minor comments.

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/data-observatory-measure-model.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        validators: ['required']
+      },
+      measurement: {
+        type: 'Select',
+        title: _t('editor.layers.analysis-form.data-observatory-measurement-measurement'),
+        options: this.doMetadata.measurement(this.get('area')).sort()
+      },
+      segments: {
+        type: 'Select',
+        title: '', // _t('editor.layers.analysis-form.data-observatory-measurement-column'),
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/data-observatory-measure-model.js]: If we don't need the comment, I'd get rid of it.

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/data-observatory-measure.tpl (hunk 2 above)
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/data-observatory-measure.tpl]: finally I removed that funcionallity

**Pull request merged**